README.md

```diff
@@ -2,55 +2,34 @@
 
 # Project Name
```

Comment:
- [ ] Please kindly update your readme file with the title of your project, kindly replace the placeholder text (`Project Name`) above with the title of your project. This will make your readme file more informative and professional.